packages/deployment/upgrade-test/upgrade-test-scripts/agoric-upgrade-11/actions.sh

```diff
@@ -1,13 +1,11 @@
 #!/bin/bash
 
-. ./upgrade-test-scripts/env_setup.sh
+SDK=${SDK:-/usr/src/agoric-sdk}
```

Comment:
please comment what environment provides that source path. Docker? and what wants to override it